src/docs.rs

```diff
@@ -19,27 +19,40 @@ pub fn build_project(
     // Read and type check project
     let (config, analysed) = project::read_and_analyse(&project_root)?;
 
-    // Get README content
-    let readme =
-        std::fs::read_to_string(project_root.as_ref().join("README.md")).unwrap_or_default();
+    // Initialize pages with the README
+    let mut pages = vec![DocsPage {
+        title: "README".to_string(),
+        path: "index.html".to_string(),
```

Comment:
Can we make this empty and then have the `index.html` be appended for all pages when rendered?